src/ui-shell/header/hamburger.component.ts

```diff
@@ -10,15 +10,45 @@ import { I18n } from "../../i18n/i18n.module";
 	selector: "ibm-hamburger",
 	template: `
 		<button
+			(click)="doClick($event)"
+			[ngClass]="{'bx--header__action--active': selected}"
 			class="bx--header__menu-trigger bx--header__action"
 			[attr.aria-label]="i18n.get('UI_SHELL.HEADER.MENU') | async"
 			[attr.title]="i18n.get('UI_SHELL.HEADER.MENU') | async">
-			<svg aria-hidden="true" width="20" height="20" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32">
+
+			<svg *ngIf="!selected" aria-hidden="true" width="20" height="20" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32">
 				<path d="M4 6h24v2H4zm0 18h24v2H4zm0-9h24v2H4z" />
 			</svg>
+
+			<svg *ngIf="selected" aria-hidden="true" height="20" width="20" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32">
+				<path d="M24 9.4L22.6 8 16 14.6 9.4 8 8 9.4l6.6 6.6L8 22.6 9.4 24l6.6-6.6 6.6 6.6 1.4-1.4-6.6-6.6L24 9.4z" />
+			</svg>
 		</button>
 	`
 })
 export class Hamburger {
+	/**
+	 * Controls the active/selected state for the `Hamburger` menu.
+	 * @type {boolean}
+	 * @memberof Hamburger
+	 */
+	@Input() selected = false;
```

Comment:
I'm going to mess everything up here **but**:

Can `selected` be the event emitter? (we _primarily_ use `selected` for events like this)

We can swap `selected` for `active` (which lines up with the class/intent a bit better anyway).